vespajlib/src/main/java/com/yahoo/tensor/TensorAddress.java

```diff
@@ -13,6 +13,21 @@
  * @author bratseth
  */
 public abstract class TensorAddress implements Comparable<TensorAddress> {
+    private static final String [] SMALL_INDEXES = createSmallIndexesAsStrings(1000);
```

Comment:
Space between `String` and `[]` looks very much like a C programmer coding Java ;)

Reply:
:)

Reply:
priceless